pages/services/prometheus/0.1.0-2.3.1/install/install.md

```diff
@@ -0,0 +1,299 @@
+
+All dcos prometheus CLI commands have a --name  argument allowing the user to specify which instance to query. If you do not specify a service name, the CLI assumes a default value matching the package name, i.e. prometheus. The default value for --name can be customized via the DC/OS CLI configuration:
+
+   ```shell
+   dcos prometheus --name=prometheus <cmd>
+   ```
+
+You can specify a custom configuration in an `options.json` file and pass it to `dcos package install` using the `--options` parameter.
+
+   ```shell
+   dcos package install prometheus --options=options.json
+   ```
+
+For more information on building the `options.json` file, see [DC/OS documentation](https://docs.mesosphere.com/latest/usage/managing-services/config-universe-service/) for service configuration access.
+
+## Installing from the DC/OS Web Interface
+
+Note:  Alternatively, you can install Prometheus from the DC/OS web interface by clicking on Deploy after selecting the app from Catalog.
+   
+If you install Prometheus from the DC/OS web interface, the 
+dcos prometheus CLI commands are not automatically installed to your workstation. They may be manually installed using the DC/OS CLI:
+
+
+   ```shell
+   dcos package install prometheus --cli
+   ```
+
+## Note : For Prometheus configuration user must be aware of all the required configuration to be mentioned under prometheus.yml
+example : if user requires service discovery he\she would need to setup the configuration under prometheus yml before running the service.
+
+Same way whatever things are not required needs to be removed from prometheus yml and required addtional configuration needs to be mentioend for any addtional features required in the service.
+
+## Installing HA-alert manager with base build :
+   
+   By default , prometheus will launch HA-prometheus server and to spin up with HA-Alert manager , alert manager check box need to be checked and respective alert manager target would need to be mentioned under prometheus yml.
+   
+## Installing HA-Prometheus without Alert Manager and point to HA-Alert Manager launched with base build:
+
+ To install HA-Prometheus without alert manager , alert manager check box not to be checked under alert manager config.
+ To point HA-Prometheus server to base\first build HA-Alert manger,we require to pass alert manager endpoint as target for each of HA-Prometheus services we run under prometheus configuration yml.
+
+## Installing HA-global prometheus 
+
+To federate data from slave prometheus to global prometheus,HA-global prometheus is requird to be installed.
```

Comment:
review and revise this section as well based on what exists in dcos-prometheus

Reply:
Also, the service guide should document how a user can configure Prometheus Federation as a pattern but that not be in basic install page.